Makefile

```diff
@@ -455,5 +455,19 @@ push-bundle-to-quay:
 .PHONY: dev-release
 ## validating the operator by installing new quay releases
 dev-release:
-	echo  "HELLO BUNDLE VERSION -${BUNDLE_VERSION}"
-	sleep 10m
+	$(Q)kubectl delete operatorsources community-operators -n=openshift-marketplace
```

Comment:
You sure the CVO operator which manages this wouldn't bring this resource back? 🤕